networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/AttestationTopicSubscriber.java

```diff
@@ -94,10 +93,10 @@ public synchronized void subscribeToPersistentSubnets(
   public synchronized void onSlot(final UnsignedLong slot) {
     boolean shouldUpdateENR = false;
 
-    final Iterator<Entry<Integer, UnsignedLong>> iterator =
+    final Iterator<Map.Entry<Integer, UnsignedLong>> iterator =
```

Comment:
Are we enforced now to specify any internal classes this way?

Reply:
Not all internal classes - error prone has a rule to prevent importing classes or methods with overly generic names.  It used to mostly hit me on `Builder` but has apparently been expanded to include `Entry` and `valueOf`.  I can live with it but it would be very easy to convince me to just disable the whole rule as I can't think of a time when it wasn't obvious which `Builder` was meant in context.